crane/config.go

```diff
@@ -31,7 +36,11 @@ type config struct {
 	RawContainerMap map[string]*container `json:"containers" yaml:"containers"`
 	RawGroups       map[string][]string   `json:"groups" yaml:"groups"`
 	RawHooksMap     map[string]hooks      `json:"hooks" yaml:"hooks"`
+	RawNetworks     map[string]*network   `json:"networks" yaml:"networks"`
```

Comment:
You're right. I decided to standardize them according to the fields in the config: e66b169cd6e2bcd90c175136f85c3e12876899dc